pypeit/utils.py

```diff
@@ -1164,6 +1200,119 @@ def robust_polyfit(xarray, yarray, order, weights=None, maxone=True, sigma=3.0,
     ct = func_fit(xfit, yfit, function, order, w=wfit, minv=minv, maxv=maxv, bspline_par=bspline_par)
     return mask, ct
 
+def robust_polyfit_djs(xarray, yarray, order, function = 'polynomial', minv = None, maxv = None, bspline_par = None,
```

Comment:
We definitely should not have two fitters in `PypeIt`.  
If we choose this one we need to remove the other.

Reply:
robust_polyfit is on the way out in the next PR to be replaced by the dis better version of it.